### PR TITLE
new appstudio limits and quotas

### PR DESCRIPTION
**Pull request by amfred**
Paired with: https://github.com/codeready-toolchain/host-operator/pull/944

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -440,10 +440,10 @@ type appstudioTierChecks struct {
 
 func (a *appstudioTierChecks) GetNamespaceObjectChecks(_ string) []namespaceObjectsCheck {
 	checks := []namespaceObjectsCheck{
-		resourceQuotaComputeDeploy("20", "32Gi", "1750m", "32Gi"),
-		resourceQuotaComputeBuild("120", "128Gi", "12", "64Gi"),
+		resourceQuotaComputeDeploy("0", "32Gi", "1750m", "32Gi"),
+		resourceQuotaComputeBuild("0", "128Gi", "12", "64Gi"),
 		resourceQuotaStorage("50Gi", "200Gi", "50Gi", "30"),
-		limitRange("2", "2Gi", "10m", "256Mi"),
+		limitRange("0", "2Gi", "10m", "256Mi"),
```
Comment (hugares) [inline testsupport/tiers/checks.go:446]: Should we put 200m instead of 10m as default cpu request as performance team tested?

Comment (amfred) [reply testsupport/tiers/checks.go]: Yes, thanks.

File: test/migration/setup_runner.go [hunk 2]
```diff
@@ -198,6 +206,21 @@ func (r *SetupMigrationRunner) prepareAppStudioProvisionedUser(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func (r *SetupMigrationRunner) prepareAppStudioProvisionedUserLarge(t *testing.T) {
+	usersignup := r.prepareUser(t, AppStudioProvisionedUserLarge, r.Awaitilities.Member1())
+	hostAwait := r.Awaitilities.Host()
+
+	// promote to appstudio
```
[the remark this answers is not in the document]
Comment (amfred) [reply test/migration/setup_runner.go:213]: Oh yeah.

**Comment by alexeykazakov** [comment]
/retest

Infra issue

**Comment by amfred** [comment]
I don't think this test failure is related to my change, and the corresponding tests on the host-operator are passing:
`
: Find all of the input images from ocp/4.14 and tag them into the stable stream expand_less | 34s
-- | --
{  could not create snapshot imagestream ocp/4.14 for release latest: Timeout: request did not complete within requested timeout - context deadline exceeded}

`

**Comment by alexeykazakov** [comment]
It looks like an openshift-ci infra issue.

**Comment by amfred** [comment]
See the comments on the paired PR; I think we're good to go.

**Comment by amfred** [comment]
@bamachrn What are the new PRs that replace this?

**Comment by MatousJobanek** [comment]
This PR hasn't been updated for a long time and the paired PR is already closed. 
I'm closing the PR, feel free to reopen it or create a new one if you still need to update the tiers

**Pull request closed without merging**